package.json

```diff
@@ -6,6 +6,7 @@
     "dependencies": {
         "@types/express": "^4.17.6",
         "@types/jquery": "^3.3.33",
+        "@types/uuid": "^7.0.3",
```

Comment:
Have to move this to dependency section to avoid the purging-devDependencies problem of heroku